test/WebJobs.Script.Tests/Binding/ActionResults/RawScriptResultTests.cs

```diff
@@ -79,7 +78,7 @@ public async Task AddsHttpCookies()
             context.HttpContext.Response.Headers.TryGetValue("Set-Cookie", out StringValues cookies);
 
             Assert.Equal(2, cookies.Count);
-            Assert.Equal("firstCookie=cookieValue; path=/", cookies[0]);
+            Assert.Equal("firstCookie=cookieValue; path=/; samesite=none", cookies[0]);
```

Comment:
This changed from 2.2 -> 3.0 -> 3.1. The value here went from `samesite=lax` to blank, to `samesite=none`... @mhoeger any idea what the right behavior is?